### PR TITLE
OSRA-397 Sponsorship totals on sponsors index

### PR DESCRIPTION
**Pull request by NikitaAvvakumov**
https://osraav.atlassian.net/browse/OSRA-397

## AA
 - add subtitle to display totals of active & requested sponsorships
 - disable batch actions globally as they are not used by the app
screen shot 2015-03-26 at 1 32 16 pm

## HQ
 - add subtitle to display totals of active & requested sponsorships
 - create view helper methods to calculate the totals (plan to reuse them later
   on a dashboard or elsewhere)
 - view spec for the index page
 - helper spec
screen shot 2015-03-26 at 1 57 10 pm

### CODE REVIEW

File: app/admin/sponsor.rb [hunk 1]
```diff
@@ -1,5 +1,19 @@
 ActiveAdmin.register Sponsor do
 
+  actions :all, except: [:destroy]
+
+  total_requested_sponsorships = -> { Sponsor.pluck(:requested_orphan_count).sum }
+  total_active_sponsorships = -> { Sponsorship.all_active.size }
```
Comment (razorcd) [inline app/admin/sponsor.rb]: AA doesn't have access to the helper files?

File: spec/views/hq/sponsors/index_spec.rb [hunk 2]
```diff
@@ -8,6 +8,19 @@
...
+      and_return 3
+    allow(view).to receive(:total_requested_sponsorships).
+      and_return 10
+
+    render
+
+    expect(rendered).
+      to have_text 'Currently sponsoring 3 orphans out of 10 requested.'
+  end
+
```
Comment (razorcd) [inline spec/views/hq/sponsors/index_spec.rb]: Any reason you are using `allow` instead of `expect`? I favor `expect`.

Comment (NikitaAvvakumov) [inline spec/views/hq/sponsors/index_spec.rb]: I lean towards the opinion I expressed in the scrum: while nothing would change in practical terms by switching to `expect`, I favour the structure that uses `allow` in the setup phase and `expect` in the verification phase. Using `expect` on L13 & 15 would be testing the implementation, while what I'd rather focus on here is the final outcome - that the right text is rendered.

File: app/admin/sponsor.rb (hunk 1 above)
Comment (NikitaAvvakumov) [inline app/admin/sponsor.rb]: Not by default, but there is a way to make it happen: http://stackoverflow.com/a/13140275/2623544. I'll push an update shortly. We have other AA helpers that duplicate HQ helper functionality - can either get rid of them too or wait until the whole of AA goes away.

**Comment by NikitaAvvakumov** [comment]
Updated to incorporate @razorcd's suggestion

**Pull request merged**